src/main/java/seedu/duke/duke/Duke.java

```diff
@@ -0,0 +1,46 @@
+package seedu.duke.duke;
+
+import java.io.IOException;
+
+import seedu.duke.ui.Ui;
+import seedu.duke.utils.Parser;
+import seedu.duke.utils.Storage;
+import seedu.duke.utils.TaskList;
+
+/**
+ * main Duke class
+ */
+public class Duke {
+    private Ui ui;
+    private Storage storage;
+    private TaskList taskList;
+    private Parser parser;
```

Comment:
Constants should be in all Caps.

Reply:
hmm thanks for this - but these arent exactly constants!

Reply:
but still thanks for the reminder! updated my timeFormat in Task.java to TIMEFORMAT